esi/lib/networks.py

```diff
@@ -112,3 +112,36 @@ def get_networks_from_port(connection, port, networks_dict={}, floating_ips_dict
         floating_network = connection.network.get_network(floating_network_id)
 
     return parent_network, trunk_networks, trunk_ports, floating_network
+
+
+def create_port(connection, network_name):
```

Comment:
I think there should also be an optional "name" argument; if it's None, then the function should generate a port name. The overall code should be similar to this:

https://github.com/CCI-MOC/esisdk/blob/main/esi/lib/nodes.py#L211-L219

And we would probably want to update that code to use this function.

We can also have the network itself be passed in, instead of the network_name.

Reply:
I simplied function. Updated it now.

Reply:
I passed only network name due to dropdown selection. Updated the code now.